portal/static/js/main.js

```diff
@@ -471,8 +476,13 @@ var fillContent = {
                         if (clinicalValue == "true") {
                             if (hasValue(val.content.issued)) {
                                 var issuedDate = "";
-                                //d M y format
-                                $("#biopsyDate").val(tnthDates.formatDateString(val.content.issued));
+                                //$("#biopsyDate").val(tnthDates.formatDateString(val.content.issued));
+                                var dString = (val.content.issued).substring(0, (val.content.issued).indexOf("T"));
```

Comment:
I think these dates are all ISO 8601, do you have a common function for parsing these?